src/platform/windows.rs

```diff
@@ -133,7 +132,7 @@ pub trait WindowBuilderExtWindows {
     /// A child window has the WS_CHILD style and is confined to the client area of its parent window.
     ///
     /// For more information, see <https://docs.microsoft.com/en-us/windows/win32/winmsg/window-features#child-windows>
-    fn with_parent_window(self, parent: HWND) -> WindowBuilder;
+    fn with_parent_window(self, parent: *mut c_void) -> WindowBuilder;
```

Comment:
It would be helpful to document the native types of these parameters (e.g. `HWND`, `HMENU`, `WORD`, `DWORD`) now that they are passed as `c_void` or rust types.

Reply:
Alternatively, adding type aliases to winit would be even better at self documenting.

Reply:
Would it make sense to define `HWND`, `HMENU`, etc. as `isize`, rather than `*mut c_void`? Since someone here pointed out that handles are not guaranteed to be pointers and windows-sys also defines them as `isize`?

Reply:
I have just implemented it like that and updated the changelog. I hope you're fine with it.